Source/UI/NewUI/CommonUI/View/TableViewCell/CellButton.swift

```diff
@@ -18,6 +18,7 @@ final class CellButton: UITableViewCell {
 
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
+        backgroundColor = .clear
```

Comment:
I am not sure this is needed?

Reply:
I have tested removing this line then background is .white. For now, Gonna keep it .clear